### PR TITLE
added workflow file for automatic release with github actions

### PR DESCRIPTION
**Pull request by Lveltmaat**

### CODE REVIEW

**Comment by Lveltmaat** [comment]
Het scriptje checkt niet of het bestandje versions bestaat of niet

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    - name: read version file 
+      id: getversion
+      run: echo "::set-output name=version::$(cat VERSION)"
+
+    - name: Build tar files
+      run: make dist
+
+    - name: create the release
+      id: create-new-release
+      uses: actions/create-release@v1
```
Comment (rtuk) [inline .github/workflows/action.yml]: This action is archived and no longer maintained

File: .github/workflows/action.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create-new-release.outputs.upload_url }}
+        asset_path:  ./dist/galera-health-${{ steps.getversion.outputs.version }}-freebsd-amd64.tar.gz
+        asset_name: galera-health-${{ steps.getversion.outputs.version }}-freebsd-amd64.tar.gz
+        asset_content_type: application/zip
+
+
```
Comment (rtuk) [inline .github/workflows/action.yml]: remove unneeded empty lines (1 final new line is sufficient)

Comment (rtuk) [reply .github/workflows/action.yml]: Remove .travis.yml

Comment (rtuk) [reply .github/workflows/action.yml]: remove tags used for testing

Comment (rtuk) [reply .github/workflows/action.yml]: remove release (the first release should happen once this PR is merged)

**Review by rtuk — CHANGES_REQUESTED** [review]

**Review by rtuk — APPROVED** [review]

**Pull request closed without merging**